tests/bootstrap.php

```diff
@@ -27,6 +27,7 @@
 $install_dir = realpath(__DIR__ . '/..');
 
 require $install_dir . '/includes/defaults.inc.php';
+require $install_dir . '/includes/definitions.inc.php';
```

Comment:
It might fail to connect but carries on still. Will double check / tidy up later.